zrml/prediction-markets/src/weights.rs

```diff
@@ -415,11 +416,14 @@ impl<T: frame_system::Config> WeightInfoZeitgeist for WeightInfo<T> {
     // Storage: MarketCommons Markets (r:1 w:1)
     // Storage: Timestamp Now (r:1 w:0)
     // Storage: PredictionMarkets MarketIdsPerReportBlock (r:1 w:1)
-    fn report(_m: u32) -> Weight {
+    fn report_market_with_dispute_mechanism(_m: u32) -> Weight {
         Weight::from_ref_time(69_185_134)
             .saturating_add(T::DbWeight::get().reads(3))
             .saturating_add(T::DbWeight::get().writes(2))
     }
+    fn report_trusted_market() -> Weight {
+        Weight::from_ref_time(123)
```

Comment:
I mean there is at least one read and write for the market. What is happening here? This can't be the real weight.

Reply:
Yeah, I didn't run any benchmarks.